src/main/java/entities/Listing.java

```diff
@@ -12,18 +12,37 @@ public class Listing {
     private static int nextId = 0;
     private LocalDate dateAdded; // Represents a date (year, month, day (yyyy-MM-dd))
     private float price;
-    private User seller;
+    private String sellerUsername;
+    private int sellerRating;
+    private static int nextID = 0;
     private String description;
-    private List<String> images; // Will store the directory path of the images
+    private String imagePath; // Will store the directory path of the images
```

Comment:
are we doing multiple images? if so we should have both image path and the array of image names

Reply:
we decided to only have one image to keep it simple

Reply:
sounds good. I will have to change my create listing class after we merge everything’s